server/src/api/accounts/accounts.service.ts

```diff
@@ -61,34 +61,32 @@ export const create = async (
     text: `
       INSERT INTO users
       VALUES (DEFAULT, $1, DEFAULT, $2)
-      RETURNING *
-      `,
+      RETURNING *;`,
```

Comment:
I mentioned it above, but you're original change was right. I'm not sure why it wasn't working for me but it is now